### DISCUSSION TITLE
Billing options

### DISCUSSION THREAD

**Discussion by catarinaguerreiro** [codemagic-ci-cd/codemagic-docs#2615] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

My doubt is not in the functionality, is on the billing plan. I created an account for my team, I am steel in the Trial plan, but if I select the 'Pay as you go' plan, can I use the 500 minutes that my account says I have free for month and, if I need more, when this 500minutes ends, I start to pay for the extra build minutes?

### Expected results

-

### Actual results

-

**Comment by icarusdust** [comment]
Hi @catarinaguerreiro, you can use 500 free build minutes only for personal accounts, not team accounts. More information [here](https://docs.codemagic.io/billing/pricing/#1-free-plan).
